builtin/logical/transit/path_keys.go

```diff
@@ -109,6 +109,9 @@ func (b *backend) pathPolicyRead(
 	if p.Derived {
 		resp.Data["kdf_mode"] = p.KDFMode
 		resp.Data["convergent_encryption"] = p.ConvergentEncryption
+		if p.ConvergentEncryption {
```

Comment:
I'll always be set, but no point in outputting it to the user if convergent encryption isn't turned on.